javascript/javascript1/week4/formatternum.js

```diff
@@ -0,0 +1,19 @@
+function formatCreditCardNumber(number) {
```

Comment:
Try to avoid generic prop, variable, function and object names, as there is something called [reserved words](https://www.w3schools.com/js/js_reserved.asp). `number` is not a reserved word, but sounds dangerously generic. Better would be to call it something as `cardNumber`